packages/cli/generators/controller/index.js

```diff
@@ -75,6 +75,33 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
   promptArtifactType() {
     debug('Prompting for controller type');
     if (this.shouldExit()) return;
+    // remind user the input might get changed if it contains _ or accents char
+    if (this.artifactInfo.name.includes('_')) {
+      this.log(
+        chalk.red('>>> ') +
+          `Underscores _ in the class name will get removed: ${
+            this.artifactInfo.name
+          }`,
+      );
+    }
+    if (this.artifactInfo.name.match(/[\u00C0-\u024F\u1E00-\u1EFF]/)) {
+      this.log(
+        chalk.red('>>> ') +
+          `Accented chars in the class name will get replaced: ${
+            this.artifactInfo.name
+          }`,
+      );
+    }
+    // inform user what controller/file names will be created
+    this.log(
+      `Controller ${chalk.yellow(
+        utils.toClassName(this.artifactInfo.name),
+      )} will be created in src/controllers/${chalk.yellow(
+        utils.toFileName(this.artifactInfo.name) + '.controller.ts',
+      )}`,
+    );
+    this.log();
+
```

Comment:
I'm seeing the prompts for a few generators. Let's refactor it into `ArtifactGenerator`.